app/models/features.rb

```diff
@@ -1,7 +1,7 @@
 # Co-erces an HStore object of feature flags into a ruby model
 class Features
   include Virtus.model
-  attribute :check_in, Boolean, default: false
+  attribute :check_in, Boolean, default: true
```

Comment:
If you want to enable the check in feature for your user, do it through the rails console:

```
a = Account.find_by(email: "the-email-you-want@to.use")
a.features.check_in = true
a.save
```